Libplanet.RocksDBStore/RocksDBStore.cs

```diff
@@ -29,6 +29,7 @@ public class RocksDBStore : BaseStore
         private const string TxIndexDbName = "txindex";
         private const string StagedTxDbName = "stagedtx";
         private const string ChainDbName = "chain";
+        private const int ForkWriteBatchSize = 100000;
```

Comment:
How about making this value configurable?

Reply:
That's possible, but this batch isn't affected by the nature of the application (only affected by internal data structure).  and there are too many configurable variables in this class 😅